crypto/ec/ecx_meth.c

```diff
@@ -35,7 +35,8 @@ typedef enum {
 
 /* Setup EVP_PKEY using public, private or generation */
 static int ecx_key_op(EVP_PKEY *pkey, int id, const X509_ALGOR *palg,
-                      const unsigned char *p, int plen, ecx_key_op_t op)
+                      const unsigned char *p, int plen, ecx_key_op_t op,
+                      OPENSSL_CTX *libctx, const char *propq)
```

Comment:
why pass propq if it is not used?

Reply:
It's prudent to keep the libctx, propq pair I suspect.
There are currently some places the property query doesn't reach down to but the libctx does.

Reply:
Which makes me to ask - wouldn't it make sense to make the propq part of the libctx instead of having to add two parameters everywhere? @mattcaswell @levitte ?

Reply:
No. We already have a global libctx wide propq. The idea is to be able to override it for specific calls.

Reply:
> It's prudent to keep the libctx, propq pair I suspect.
There are currently some places the property query doesn't reach down to but the libctx does.

Right. I think in general we should always be passing the pair. We might have to go back and fix a few spots where we haven't done that at some point